ipynb/primary-processing/5-alpha_diversity.md

```diff
@@ -0,0 +1,71 @@
+In this next chapter, we're going to calculate alpha diversity over all of the samples in the meta-analysis. As always, let's first get our environment setup.
+
+```python
+>>> import os
+...
+>>> import americangut.notebook_environment as agenv
+>>> import americangut.util as agu
+...
+>>> import qiime_default_reference as qdr
+...
+>>> chp_path = agenv.activate('5')
+>>> greengenes_tree = qdr.get_reference_tree()
+>>> cpu_count = agenv.get_cpu_count()
+```
+
+And then we'll pull in the files we need for processing.
+
+```python
+>>> ag_pgp_hmp_gg_100nt_biom = agu.get_existing_path(agenv.paths['ag-pgp-hmp-gg-100nt-biom'])
+>>> ag_gg_100nt_biom         = agu.get_existing_path(agenv.paths['ag-gg-100nt-biom'])
+>>> ag_pgp_hmp_gg_cleaned_md = agu.get_existing_path(agenv.paths['ag-pgp-hmp-gg-cleaned-md'])
+```
+
+As well as establish the new paths that we'll be creating.
+
+```python
+>>> ag_pgp_hmp_gg_100nt_1k_multi   = agu.get_new_path(agenv.paths['ag-pgp-hmp-gg-100nt-1k-multiple'])
```

Comment:
@jwdebelius, I've got a plan on how to make this work for multiple
rarefaction levels, with at least some code already on place. I'm going to
work on it while these notebooks get reviewed/merged in parallel as I think
it's important to get the basic bits merged in
On Oct 22, 2015 1:35 PM, "J W Debelius" <notifications@github.com> wrote:

> In ipynb/primary-processing/5-alpha_diversity.md
> <https://github.com/biocore/American-Gut/pull/166#discussion_r42802502>:
>
> > +>>> greengenes_tree = qdr.get_reference_tree()
> > +>>> cpu_count = agenv.get_cpu_count()
> > +```
> > +
> > +And then we'll pull in the files we need for processing.
> > +
> > +```python
> > +>>> ag_pgp_hmp_gg_100nt_biom = agu.get_existing_path(agenv.paths['ag-pgp-hmp-gg-100nt-biom'])
> > +>>> ag_gg_100nt_biom         = agu.get_existing_path(agenv.paths['ag-gg-100nt-biom'])
> > +>>> ag_pgp_hmp_gg_cleaned_md = agu.get_existing_path(agenv.paths['ag-pgp-hmp-gg-cleaned-md'])
> > +```
> > +
> > +As well as establish the new paths that we'll be creating.
> > +
> > +```python
> > +>>> ag_pgp_hmp_gg_100nt_1k_multi   = agu.get_new_path(agenv.paths['ag-pgp-hmp-gg-100nt-1k-multiple'])
>
> Ok, not really blocking.
>
> I will also look at adding the single bodysite and single sample from the
> preprocessing notebook, if thats desired here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/American-Gut/pull/166/files#r42802502>.
>